tests/cypress/support/welcome_and_header.js

```diff
@@ -177,7 +181,11 @@ export const userMenu = {
         cy.waitUntil(() => {
             return cy.get('[data-test="app-dropdown"]').should('be.visible')
         }, {'interval': 1000, 'timeout':120000})
-        cy.get('[data-test="app-dropdown"]').should('be.visible').click()
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.get('[data-test="app-dropdown"]').should('be.visible').wait(500).click().wait(500)
```

Comment:
I found in some cases, cypress force click will pass the current step but not actually click the button and failed in the later step, this is a good example that can repeatedly prove this scenario

<img width="1599" alt="Screen Shot 2021-10-08 at 10 03 16 AM" src="https://user-images.githubusercontent.com/55596988/137327542-8881d8c5-ce12-456e-9c77-f32577cc914b.png">